_stbt/transition.py

```diff
@@ -30,7 +37,8 @@ def press_and_wait(
     region: Region = Region.ALL,
     timeout_secs: float = 10,
     stable_secs: float = 1,
-    min_size: SizeT = None,
+    min_size: "SizeT|None" = None,
+    frames: "typing.Iterator[core.Frame]|None" = None,
     _dut=None,
```

Comment:
I see, `press_and_wait` still needs `_dut` for `_dut.press()` (for the unit tests). At least we removed `_dut` from `wait_for_transition_to_end`. :+1: